tests/test_model_functions.py

```diff
@@ -260,9 +258,9 @@ def test_add_unit(self, model):
             model.add_unit('unlikely_unit_name', [{'units': 'millivolt'}], base_units=True)
         assert len(model.units.custom_defined) == 6
 
-    def test_add_variable(self, model):
+    def test_add_variable(self, local_model):
         """ Tests the Model.add_variable() method. """
-
+        model = local_model
```

Comment:
Are we allowed modify this model?

Also, might be good to have a shared model loading method somewhere, e.g. in `conftest` or some other shared file?

Reply:
Yes we can modify the local version
Added shared.load_model